src/main/java/teammates/common/datatransfer/FeedbackQuestionAttributes.java

```diff
@@ -568,4 +582,19 @@ public int getNumberOfEntitiesToGiveFeedbackTo() {
     public String getQuestionAdditionalInfoHtml() {
         return getQuestionDetails().getQuestionAdditionalInfoHtml(questionNumber, "");
     }
+    
+    /**
+     * Should only be used for testing
+     */
+    public void setCreated_NonProduction(Date createdAt) {
+        this.createdAt = createdAt;
+    }
+    
+    /**
+     * Should only be used for testing
+     */
+    public void setUpdatedAt_NonProduction(Date updatedAt) {
```

Comment:
Actually, is there a reason why it's `setCreated` instead of `setCreatedAt`?
The latter seems more fitting, considering the rest are in that format.
@LowWeiLin too.